src/esp/bindings_js/index.js

```diff
@@ -23,7 +23,7 @@ function preload(url) {
   if (url.indexOf("http") === -1) {
     let file_parents = splits.slice(0, splits.length - 1);
     for (let i = 0; i < splits.length - 1; i += 1) {
-      file_parents_str += file_parents[i] + "/";
+      file_parents_str += file_parents[i];
```

Comment:
This was not failing on my local Ubuntu but was failing for CI (FS runtime error), maybe due to differences in emscripten version or browser.

Reply:
Hold on, this will break any assets in subfolders if you make this change. (It will work if the asset is not in  subfolder because the splits is 0. Are you sure the physics JSON is copied correctly during the Build Process? Will assets in subfolder still work? I don't think we actually test that in the CI so that breakage wouldn't be detected by this "fix". Won't this just instead create a single folder that mashes together the asset path names? Ie: dataphysicsconfig.json instead of data/physicsconfig.json?